scm/webhook.go

```diff
@@ -37,6 +37,14 @@ type (
 		Commits []Commit
 	}
 
+	// PipelineHook
+	PipelineHook struct {
+		Repo     Repository
+		Commit   Commit
```

Comment:
should we include `PullRequest` in case pipeline runs for a pull request?

Reply:
Added the pull request